.github/workflows/algolia-scraper.yml

```diff
@@ -0,0 +1,31 @@
+name: algolia-scraper
```

Comment:
This needs to get added to the other deployment.yml file that we just merged, so it happens as part of a deployment.

Reply:
so is there any changes for me? I don't think i have access to the deployment.yml at this time

Reply:
Rebase, and pick up https://github.com/Seneca-ICTOER/IPC144/blob/main/.github/workflows/deployment.yaml

Reply:
ok